pkg/auth/handler/webapp/viewmodels/base.go

```diff
@@ -257,6 +267,23 @@ func (m *BaseViewModeler) ViewModel(r *http.Request, rw http.ResponseWriter) Bas
 		GoogleTagManagerContainerID: m.GoogleTagManager.ContainerID,
 		HasThirdPartyClient:         hasThirdPartyApp,
 		AuthUISentryDSN:             string(m.AuthUISentryDSN),
+		LogUnknownError: func(err map[string]interface{}) string {
+			if err != nil {
+				var sessionID string
+				if s := webapp.GetSession(r.Context()); s != nil {
+					sessionID = s.ID
+				}
+
+				m.Logger.WithFields(map[string]interface{}{
+					"session_id":  sessionID,
+					"request_uri": r.URL.RequestURI(),
+				}).Errorf("unknown error: %v", err)
```

Comment:
`err` here is the JSON serialization of APIError. So we should log all fields of it. Remove `session_id` and `request_uri`.

```
m.Logger.WithFields(err).Errorf("unknown error: %v", err)
```